internal/restic/lock.go

```diff
@@ -360,6 +360,12 @@ func ForAllLocks(ctx context.Context, repo Repository, excludeID *ID, fn func(ID
 		if excludeID != nil && id.Equal(*excludeID) {
 			return nil
 		}
+		if size == 0 {
```

Comment:
I would also be possible to just ignore an error when loading a lock file if it cannot be loaded. However, that would still cause the "invalid data" warning to be printed.